azure-devops/azext_devops/dev/boards/arguments.py

```diff
@@ -2,7 +2,9 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-from azure.cli.core.commands.parameters import get_three_state_flag
+
+from azure.cli.core.commands.parameters import get_enum_type, get_three_state_flag
+_EXPAND_TYPES = ['none', 'relations', 'fields', 'links', 'all']
```

Comment:
It will much better if this is coming from a class in python client.
I already see this issue being raised once https://github.com/microsoft/azure-devops-python-api/issues/31 but can't find the class in current version

Reply:
I agree, but many classes are not available in the client